src/App.css

```diff
@@ -1,5 +1,6 @@
 .App {
   text-align: center;
   width: 100vw;
-  min-height: 100vh;
+  min-height: 90vh;
+  background-color: white;
```

Comment:
Please revert these changes. The last line conflicts with the theme's background color

Reply:
Ok removed them from there